internal/layers/layers.go

```diff
@@ -286,6 +300,17 @@ func UnmountContainerLayers(ctx context.Context, layerFolders []string, containe
 		if len(layerFolders) < 1 {
 			return errors.New("need at least one layer for Unmount")
 		}
+
+		// Remove the mount point if there is one. This is the case for job containers.
+		if volumeMountPath != "" {
+			if err := removeSandboxMountPoint(ctx, volumeMountPath); err != nil {
+				return err
+			}
+			if err := os.Remove(volumeMountPath); err != nil {
+				return errors.Wrapf(err, "failed to remove sandbox mounted folder path %q", volumeMountPath)
+			}
```

Comment:
isn't os.Remove called in `removeSandboxMountPoint` already?

Reply:
Yes :), removing.